lighthouse-core/audits/accessibility/duplicate-id-active.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensures every `id` attribute value of active, focusable elements is unique.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that checks if there are any duplicate id HTML attributes on active, focusable elements. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: '`[id]` attributes on active, focusable elements are unique',
+  /** Title of an accesibility audit that checks if there are any duplicate id HTML attributes on active, focusable elements. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
+  failureTitle: '`[id]` attributes on active, focusable elements are not unique',
+  /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'The value of an id attribute on focusable elements must be unique to prevent other instances from being overlooked by assistive technologies. [Learn more](https://dequeuniversity.com/rules/axe/3.3/duplicate-id-active).',
```

Comment:
nit: 'prevent from being overlooked' felt a tad awkward.

how do you feel about flipping it?

```suggestion
  description: 'The value of the `id` attribute on focusable elements must be unique to ensure all instances are visible to assistive technologies. [Learn more](https://dequeuniversity.com/rules/axe/3.3/duplicate-id-active).',
```

Reply:
I also see now that this is the exact text of the linked doc, so I guess if it's good enough for axe it's good enough for me 🤷‍♂

Reply:
How about "All focusable elements must have a unique ID to ensure that they're visible to assistive technologies."